README.md

```diff
@@ -28,148 +28,191 @@ inference.
     sudo apt-get install python3-pip python3-venv
     ```
 
-3. Create and activate virtual environment:
-    ```
-    python3 -m venv venv
-    source venv/bin/activate
-    ```
+3. Search for available scripts that create python virtual environments for different task types:
+   ```bash
+   find external/ -name init_venv.sh
+   ```
+   
+   Approximate output:
+   ```
+   external/mmdetection/init_venv.sh
+   external/mmsegmentation/init_venv.sh
+   external/deep-object-reid/init_venv.sh
+   ```
+   
+4. Let's create, activate Object Detection virtual environment and install `ote_cli`:
+   ```
+   ./external/mmdetection/init_venv.sh det_venv
+   source det_venv/bin/activate
+   pip3 install -e ote_cli/
+   ```
+   
+### OTE CLI tools
+1. Have a look at model templates available for this virtual environment:
+   ```
+   ote_find --root ./external/mmdetection/
+   ```
+   
+   Approximate output:
+   ```
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_VFNet
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_resnet50_VFNet/template.yaml
+     task_type: DETECTION
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_SSD
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_SSD/template.yaml
+     task_type: DETECTION
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_ATSS
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml
+     task_type: DETECTION
+   - ...
+   ```
+   Let's choose `./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml`
 
-4. Install `ote_cli` package:
-    ```
-    pip3 install -e ote_cli/
-    ```
+2. Let's have a look at `ote_train` help. These parameters are the same for all model templates.
+   ```
+   ote_train ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --help
+   ```
+   
+   Approximate output:
+   ```
+   ote_train ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --help
```

Comment:
Would it make sense to allow the user to provide either the path or the name/id?

Reply:
Will add next time